### ISSUE TITLE
Investigate an Alt-text generator service

### ISSUE THREAD

**Issue by ajkiessl** [psu-libraries/pdf_accessibility_api#43]
Kind of like how we want a GUI for users to upload PDFs to test the remediation application, we'd like a GUI to upload images and run them through the LLM to test out its alt text generation capabilities.  We could use this to help refine the LLM prompts, or choose the best LLM.  Then, if possible, find a way to easily deploy these optimizations to the production PDF Accessibility application.

**Comment by ajkiessl** [comment]
We've converted the standalone alt text generation tool to ruby: https://github.com/psu-libraries/alt_text

If we convert this to a gem, it should be easy to load it into the pdf_accessibility_api, and add a GUI for interfacing with it.
